### PR TITLE
Cluster edits - Client Cfg & Ref

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

* Moves the new CaaSP content out of the VHM chapter, and into its own new chapter in the Client Cfg Guide (with the example file beneath it)
* The VHM CaaSP chapter still exists as it was previously, with a note that you can now manage CaaSP directly instead.
* Adds new Clusters section to the Ref Guide to marry up with the UI (and associated CSH)

# Target branches

To what product version this document applies?

- [ ] Manager-3.2
- [ ] Manager-4.0
- [x] Manager-4.1
- [x] Uyuni

# Links

Fixes:

- https://bugzilla.suse.com/show_bug.cgi?id=1172932
- https://github.com/SUSE/spacewalk/issues/11730

### CODE REVIEW

File: modules/reference/pages/clm/clm-menu.adoc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-= Content Lifecycle Managment
+= Content Lifecycle Management
```
Comment (Loquacity) [inline modules/reference/pages/clm/clm-menu.adoc:1]: Drive-by typo fix :joy:

**Comment by Loquacity** [comment]
[suse_manager_client-configuration_guide.pdf](https://github.com/uyuni-project/uyuni-docs/files/4802527/suse_manager_client-configuration_guide.pdf)
[suse_manager_reference_guide.pdf](https://github.com/uyuni-project/uyuni-docs/files/4802528/suse_manager_reference_guide.pdf)

File: modules/client-configuration/pages/vhm-caasp.adoc [hunk 2]
```diff
@@ -1,12 +1,19 @@
 [[vhm-caasp]]
-= VHM and SUSE CaaS Platform
+= VHM and {caasp}
 
-You can use a {productname} VHM to gather instances from {caasp}.
+You can use a {productname} VHM to manage {caasp} clusters.
```
Comment (keichwa) [inline modules/client-configuration/pages/vhm-caasp.adoc:5]:
```suggestion
```

File: modules/client-configuration/pages/vhm-caasp.adoc [hunk 3]
```diff
@@ -1,12 +1,19 @@
...
 
+[NOTE]
+====
+You can also manage {caasp} clusters directly with {productname}, without using a VHM.
+For more information, see xref:client-configuration:clusters.adoc[].
+====
+
+
 
 == Onboarding CaaSP nodes
```
Comment (keichwa) [inline modules/client-configuration/pages/vhm-caasp.adoc:18]:
```suggestion
== Onboarding CaaSP Nodes
```
drive-by edit :)

File: modules/reference/pages/clusters/clusters-add.adoc [hunk 4]
```diff
@@ -0,0 +1,7 @@
+= Add Cluster
+
+The menu:Clusters[Add] section allows you to add new clusters to your {productname} Server.
+
```
Comment (keichwa) [inline modules/reference/pages/clusters/clusters-add.adoc:4]:
```suggestion
```

File: modules/reference/pages/clusters/clusters-menu.adoc [hunk 5]
```diff
@@ -0,0 +1,5 @@
+= Clusters Menu
+
+In the [guimenu]``Clusters`` section, you can add and manage your {caasp} clusters.
+
```
Comment (keichwa) [inline modules/reference/pages/clusters/clusters-menu.adoc:4]:
```suggestion
```

File: modules/reference/pages/clusters/clusters-overview.adoc [hunk 6]
```diff
@@ -0,0 +1,8 @@
+= Clusters Overview
+
+The menu:Clusters[Overview] section displays a list of all current clusters in your organization.
+
```
Comment (keichwa) [inline modules/reference/pages/clusters/clusters-overview.adoc:4]:
```suggestion
```

File: modules/client-configuration/pages/vhm-caasp.adoc (hunk 3 above)
Comment (Loquacity) [reply modules/client-configuration/pages/vhm-caasp.adoc:18]: :joy:
